### ISSUE TITLE
Error when add route property to +config.ts

### ISSUE THREAD

**Issue by royalswe** [brillout/vite-plugin-ssr#874]
### Description

I removed the +route.ts and added route: '/*' to the +config.ts and stumbled on this error

```
Error: [vite-plugin-ssr@0.4.123][Bug] You stumbled upon a bug in vite-plugin-ssr's source code. Go to https://github.com/brillout/vite-plugin-ssr/issues/new and copy-paste this error. (The error's stack trace is usually enough to fix the problem). A maintainer will fix the bug (usually under 24 hours). Don't hesitate to reach out as it makes vite-plugin-ssr more robust.
    at C:\Users\royni\source\repos\Sodra.Common.EpiServer.Website\Sodra.EK.WebSite.Web\WebUI\node_modules\vite-plugin-ssr\dist\cjs\shared\getPageFiles\assertPageConfigs.js:45:32
    at Array.forEach (<anonymous>)
    at assertConfigElements (C:\Users\royni\source\repos\Sodra.Common.EpiServer.Website\Sodra.EK.WebSite.Web\WebUI\node_modules\vite-plugin-ssr\dist\cjs\shared\getPageFiles\assertPageConfigs.js:29:36)
    at C:\Users\royni\source\repos\Sodra.Common.EpiServer.Website\Sodra.EK.WebSite.Web\WebUI\node_modules\vite-plugin-ssr\dist\cjs\shared\getPageFiles\assertPageConfigs.js:19:9
    at Array.forEach (<anonymous>)
    at assertPageConfigs (C:\Users\royni\source\repos\Sodra.Common.EpiServer.Website\Sodra.EK.WebSite.Web\WebUI\node_modules\vite-plugin-ssr\dist\cjs\shared\getPageFiles\assertPageConfigs.js:10:17)
    at parseGlobResults (C:\Users\royni\source\repos\Sodra.Common.EpiServer.Website\Sodra.EK.WebSite.Web\WebUI\node_modules\vite-plugin-ssr\dist\cjs\shared\getPageFiles\parseGlobResults.js:32:47)
    at setPageFiles (C:\Users\royni\source\repos\Sodra.Common.EpiServer.Website\Sodra.EK.WebSite.Web\WebUI\node_modules\vite-plugin-ssr\dist\cjs\shared\getPageFiles\setPageFiles.js:9:98)
    at globalObject.pageFilesGetter (C:\Users\royni\source\repos\Sodra.Common.EpiServer.Website\Sodra.EK.WebSite.Web\WebUI\node_modules\vite-plugin-ssr\dist\cjs\shared\getPageFiles\setPageFiles.js:17:9)
    at async getPageFilesAll (C:\Users\royni\source\repos\Sodra.Common.EpiServer.Website\Sodra.EK.WebSite.Web\WebUI\node_modules\vite-plugin-ssr\dist\cjs\shared\getPageFiles\setPageFiles.js:32:13)
    at async getRenderContext (C:\Users\royni\source\repos\Sodra.Common.EpiServer.Website\Sodra.EK.WebSite.Web\WebUI\node_modules\vite-plugin-ssr\dist\cjs\node\runtime\renderPage\renderPageContext.js:144:73)
    at async renderPage (C:\Users\royni\source\repos\Sodra.Common.EpiServer.Website\Sodra.EK.WebSite.Web\WebUI\node_modules\vite-plugin-ssr\dist\cjs\node\runtime\renderPage.js:25:25)
    at async C:\Users\royni\source\repos\Sodra.Common.EpiServer.Website\Sodra.EK.WebSite.Web\WebUI\node_modules\vite-plugin-ssr\dist\cjs\node\plugin\shared\addSsrMiddleware.js:31:27

```

The highlighted file in the pic is the one I added route property.
![image](https://github.com/brillout/vite-plugin-ssr/assets/6486940/cda5bcf6-8f57-4067-8e07-b92389083374)

**Comment by brillout** [comment]
If you restart the app do you still get the error?

**Comment by royalswe** [comment]
Yes the error is still there.

**Comment by brillout** [comment]
Can you publish the code? Or invite me to your private repo if that's an option. Or reproduction.

**Comment by royalswe** [comment]
I can create a Github repo tomorrow with the issue.

**Comment by brillout** [comment]
Neat, looking forward to it.

**Comment by royalswe** [comment]
Here you go
https://github.com/royalswe/vite-plugin-ssr_vue-server-v1/tree/main/router-error

**Comment by brillout** [comment]
Neat thanks. I'm trying to run it but I get:

```
gitpod /workspace/vite-plugin-ssr_vue-server-v1/router-error (main) $ npm i && npm run dev

added 328 packages, and audited 335 packages in 8s

58 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npm notice 
npm notice New minor version of npm available! 9.5.1 -> 9.6.6
npm notice Changelog: https://github.com/npm/cli/releases/tag/v9.6.6
npm notice Run npm install -g npm@9.6.6 to update!
npm notice 

> sodra-website@1.0.0-beta.1 dev
> vite

The list of generated files:
/home/gitpod/.vite-plugin-mkcert/dev.pem
/home/gitpod/.vite-plugin-mkcert/cert.pem
error when starting dev server:
Error: getaddrinfo ENOTFOUND localdev.sodra.com
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:107:26)
```

**Comment by royalswe** [comment]
I removed that part just now, there is allot of junk in the repo :)

**Comment by brillout** [comment]
Fix released in `0.4.123-draft.47582`. (I'll release a stable version in the next coming days.)

Thanks for the bug report!

**Comment by royalswe** [comment]
Tested the release and it Works.
